### DISCUSSION TITLE
Served from cached with high processing time.

### DISCUSSION THREAD

**Discussion by zer09** [AdguardTeam/AdGuardHome#4706] [Q&A]
I got this some entries on the query log that the processing time is greater than 50ms but it was served from cache.
![Screenshot_20220703_063118](https://user-images.githubusercontent.com/780780/177017993-55ed8ff7-d1f0-42a9-8a2a-e87acec67fe8.png).

What happening here is it normal?
I still have sub ms processing time, but there also a lot of greater than 50ms served from cache.

**Comment by ainar-g** [comment]
Could this be a CPU or i/o spike on the machine?
